ml_server/main.py

```diff
@@ -0,0 +1,79 @@
+import os
+from pathlib import Path
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from typing import List
+from functools import lru_cache
+
+from fastapi import Depends
+from gliner import GLiNER
+
+from tenant_dependency import get_tenant_info
+
+KEYCLOAK_HOST = os.getenv("KEYCLOAK_HOST", "")
+TENANT = get_tenant_info(url=KEYCLOAK_HOST, algorithm="RS256")
+ROOT_PATH = os.environ.get("ROOT_PATH", default="")
```

Comment:
Worth adding .env.example file for this service
And also to example in the root of the project

Reply:
Then this file will contain only one entry, with KEYCLOAK_HOST. 
And it's already in .env.example in the root of the project. 
And ROOT_PATH is always service-specific. So you already have it set-up in compose file, and if you don't use compose and want to just run this service, it will still work, because you won't have proxy.